src/code-examples/dialog/html.js

```diff
@@ -1,8 +1,8 @@
 const html = `<button type="button" class="js-open-modal" aria-expanded="true" aria-controls="dialog-content">Open a Dialog</button>
 
-<div id="dialog-content" class="dialog" aria-hidden="false" role="dialog" aria-labelledBy="h2" aria-modal="true">
+<div id="dialog-content" class="dialog" role="dialog" aria-labelledBy="dialog-heading" aria-modal="true">
```

Comment:
`aria-labelledBy` should be `aria-labelledby` here. We should check for other camelCase attributes–those aren't really a thing in HTML. You might see that in React code, but that's only to avoid dashes.

Reply:
Jeeze: i wrote it in a message with the lowercase "b", and then came over here and capitalized it! Fixed now: https://github.com/sparkbox/accessible-components/pull/134/commits/fdc24e64505e5ce45cfd373ac47afc33f7608563